frontstage/controllers/oauth_controller.py

```diff
@@ -29,6 +29,12 @@ def sign_in(username, password):
 
     try:
         response.raise_for_status()
+        # This if statement will only be true if the 'oauth' service is the ras-rm-auth-service.  This is only
```

Comment:
Could we formalise this in a card? Don't want this getting lost :)

Reply:
https://trello.com/c/3SYB9bV2/1078-tidy-up-api-in-ras-rm-auth-service Done!